db/migrate/20160209130938_add_service_order_and_user_name_to_miq_request.rb

```diff
@@ -0,0 +1,6 @@
+class AddServiceOrderAndUserNameToMiqRequest < ActiveRecord::Migration
+  def change
+    add_column :miq_requests, :service_order_id, :bigint
+    add_column :miq_requests, :user_name, :string
```

Comment:
@lpichler, we already have userid column and requester_name in miq_requests, I believe @gmcculloug is suggesting adding user_name to service_orders.